src/Approximations/Approximations.jl

```diff
@@ -21,6 +21,17 @@ export approximate,
        BoxDiagDirections,
        OctDirections
 
+# see also compat.jl in LazySets module
```

Comment:
Can you not say something like `import..compat.jl`?

Reply:
i'll check and merge afterwards if it works

Reply:
done